src/store/relationship.js

```diff
@@ -1,25 +1,36 @@
 import Ember from 'ember';
 import Relationship from 'ember-graph/relationship/relationship';
+import RelationshipHash from 'ember-graph/relationship/relationship_hash';
 
 const { CLIENT_STATE, SERVER_STATE, DELETED_STATE } = Relationship;
 
+// ### TODO: 1. incorporate both type AND id into hash (reduce repetitive verifications)
+// ### TODO: 2. move all remaining type== id== tests to relationship/relationships.js
```

Comment:
- [TODO 1 In github issue #97 ](https://github.com/stingerlabs/ember-graph/issues/97)
- [TODO 2 In github issue #98 ](https://github.com/stingerlabs/ember-graph/issues/98)